doc/Developing-Step-By-Step-Core.md

```diff
@@ -1941,6 +1941,7 @@ editing people:
     {
         public class EditPersonModalViewModel : GetPersonForEditOutput
         {
+            [AutoMapFrom(typeof(GetPersonForEditOutput))]
```

Comment:
Adding a mapping to constructor does nothing. Did you want to add to the `EditPersonModalViewModel` class?

Reply:
Oh, my mistake, yes, please add this annotation to the class

Reply:
@QuantaVn you can change it and commit on this PR. Is it possible?

Reply:
@alirizaadiyahsi I've updated, please review and merge